src/components/heading/Heading.stories.mdx

```diff
@@ -11,10 +11,10 @@ import {
     DEFAULT_HEADING_WEIGHT,
     DEFAULT_HEADING_MARGIN,
 } from './Heading.constants';
-import Grid from '../../foundations/layout';
+import Grid from '../../utilities/layout';
 import Text from '../text';
 
-<Meta component={Heading} title="Components/Heading" />
+<Meta component={HeadingRoot} title="Components/Heading" />
```

Comment:
why don't we use the Heading wrapper here?

Reply:
we did, the default import was just still named as `HeadingRoot`